src/main/java/appeng/helpers/DualityInterface.java

```diff
@@ -553,17 +548,20 @@ public DimensionalCoord getLocation()
 		return new DimensionalCoord( this.iHost.getTileEntity() );
 	}
 
-	public IInventory getInternalInventory()
+	public IItemHandlerModifiable getInternalInventory()
 	{
 		return this.storage;
 	}
 
 	public void markDirty()
 	{
-		for( int slot = 0; slot < this.storage.getSizeInventory(); slot++ )
-		{
-			this.onChangeInventory( this.storage, slot, InvOperation.markDirty, ItemStack.EMPTY, ItemStack.EMPTY );
-		}
+		// TODO: should not be needed
+		/*
+		 * for( int slot = 0; slot < this.storage.getSlots(); slot++ )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)